runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -38,44 +38,61 @@ vim9script
 # The communication with gdb uses GDB/MI.  See:
 # https://sourceware.org/gdb/current/onlinedocs/gdb/GDB_002fMI.html
 
-# In case this gets sourced twice.
+def Echoerr(msg: string)
+  echohl ErrorMsg | echom $'[termdebug] {msg}' | echohl None
+enddef
+
+if !has('vim9script') ||  v:version < 900
+    # Needs Vim version 9.0 and above
+    Echoerr("You need at least Vim 9.0")
+    finish
+endif
+
+# Variables to keep their status among multiple instanced of Termdebug
+# Avoid to source the script twice.
 if exists('g:termdebug_loaded')
```

Comment:
```suggestion
if g:->get('termdebug_loaded', false)
```

Reply:
Hum, could you please elaborate why that is different? To be the devil's advocate I would say that the former is more readable :p

Reply:
This is sensitive to the value of `termdebug_loaded`, whereas `exists()` is not. Either way works as long as you don't care about the value. I'd say the latter makes more sense:)